### ISSUE TITLE
Change non-standard concept_id to standard concept_id

### ISSUE THREAD

**Issue by MelaniePhilofsky** [OHDSI/Vocabulary-v5.0#1066]
# Instructions

**0. Create the initial issue.**

Per multiple community discussions and Themis conventions, concept_id = [44814659](https://athena.ohdsi.org/search-terms/terms/44814659), Multiple race needs to be updated from non-standard to standard concept.

**1. Read the documentation.**

Please familiarize yourself with the documentation [here](https://github.com/OHDSI/Vocabulary-v5.0/wiki/Community-contribution) prior to initiating the submission. **Done**

**2. Choose the category of submission.**

Choose the category of your submission and download the corresponding template:

- Template 7  **Done**

**3. Fill the template.**

Please dowload the corresponding template **Done**

Make sure you filled the checklist! **There isn't a checklist tab on the template**

**4. Upload the template.**

Please go to [this folder](https://drive.google.com/drive/u/1/folders/1rWPvto3zdS7F4QrhkpL3JXzDfm60tGOa), create a subfolder with your contribution (name it as GitHubIssueNumber_ContributionName_YourName) and upload your contribution there. **Done**

**5. Reference the template in this issue.**

Please add the link to your submission to this issue. [LINK](https://drive.google.com/drive/u/1/folders/12G_rcd9C4v1WfnW0aLIpb4QeZJHfxt02)

**6. Get the feedback.**

Please see Themis' ratified convention [here](https://ohdsi.github.io/Themis/multiple_races_per_person.html).

**Comment by aostropolets** [comment]
It seems that you want something with domain Race rather than Observation. I don't think we want to promote PCORNet concepts since it's an abandoned vocabulary but rather want to add a new concept that says Multiple race to Race domain (also thinking long-term here: if we expect an influx of other races there should be a common place to add them).

**Comment by MelaniePhilofsky** [comment]
Yes, we need to have a standard concept_id with domain_id = 'Race' and a concept_name or meaning = 'Multiple race'.

RE: vocabulary for the multiple race concept and other race concepts to be added via community contribution, some contributions might come from national / local code systems and some might come from non-coded, source value sets which are generally categorical with a data type of string text. I suggested the multiple race concept_id from PCORNet because I know some are already sites (and other CDM's, networks) are using it. The fact that the vocabulary isn't supported any more isn't a concern from the Themis point of view since this concept_id doesn't contain any standard to non-standard or hierarchical mappings. And won't contain any hierarchical mappings in the future.

Thanks for your help, @aostropolets!

**Comment by Alexdavv** [comment]
> ...and other race concepts to be added via community contribution, some contributions might come from national / local code systems and some might come from non-coded, source value sets which are generally categorical with a data type of string text. I suggested the multiple race concept_id from PCORNet because I know some are already sites (and other CDM's, networks) are using it.

We are happy to accept all these contributions, but concepts from multiple vocabularies (especially local) shouldn't themselves be promoted to Standard concepts. The reason for that is the fact that other users won't know which of multiple OMOP vocabularies contribute to the Race Domain, even though some people know "PCORNet" contains some good races. And since nobody downloads the entire set of 156, they'll end up creating their own Standard duplicates. You can argue that these vocabularies can be added to the Atnena default pre-selection, but we usually do that only for small and Domain-specific vocabularies, like "UB04 Typ bill" or "ABMS". If one contains other stuff, we don't want them to overwhelm every user's OMOP instance. The proper solution here is the creation of Standard counterparts in the "Race" vocabulary together with mappings from the source concepts.

**Comment by cgreich** [comment]
Correct, except this is probably an impossible fate. Races are not defined by any objective criteria (there are mostly socio-economical constructs with some loosely defined biological features), which only work inside a country. Therefore, it is not possible to dedup them. So we decided to open up the space. Everything goes.

**Comment by Alexdavv** [comment]
> Correct, except this is probably an impossible fate. Races are not defined by any objective criteria (there are mostly socio-economical constructs with some loosely defined biological features), which only work inside a country. Therefore, it is not possible to dedup them. So we decided to open up the space. Everything goes.

Absolutely right. But you don't want to get one loosely defined “Eastern European” from each of OMOP’s datasets. Now the mapping is arbitrary to the fact that “Race” vocabulary has some and doesn't have some others. In the future we either map to the extent people want/can OR don't map at all and the “Race” vocabulary should be gone because everybody will have their own set of races. But we can’t do a Frankenstein of both solutions.

**Comment by MelaniePhilofsky** [comment]
@aostropolets 

We would like to update our ETL in order to utilize multiple race data for a person. Will concept_id = [44814659](https://athena.ohdsi.org/search-terms/terms/44814659) be updated to standard concept with a domain_id = Race? Or is the vocabulary team going to create a concept with meaning of multiple race?

**Comment by MelaniePhilofsky** [comment]
Help! I can't find the standard concept_id for multiple race in the Race domain.

**Comment by cgreich** [comment]
Wait. Didn't we want to do a vocabularathon with all races and ethnicities?

**Comment by MelaniePhilofsky** [comment]
Multiple race does not need a vocabularathon to create a new concept or make a non-standard concept a standard concept. The ask for this has been well recorded in the forums for many years. The documentation on when and how to use this concept_id is documented in the Themis convention library. Albeit was a bit pre-mature, but the ETL folks need lead time to plan, design, build and test before deploying a big change, which we thought would go live with the Feb 2025 vocabulary release.

The impetus for modifying the conventions for race_concept_id was to house the idea of a person identifying as > 1 race or 'multiple race'. When someone brings an issue to the Themis WG, we gather all the use cases to ensure we have a comprehensive convention. Adding additional standard concept_ids for all the race values, moving >1 race  to the Observation table, enabling provenance and recording the date the data were recorded are all additional use cases the community brought forth. Not only is multiple race one of these, but it is the most requested and loudest use case.

Having a standard concept_id for multiple race and allowing for > 1 race per person in the OMOP CDM helps eliminate the following potential biases: granularity bias, representational bias, omitted variable bias, and forced choice bias.

Collaborators have been knocking on my door since the OHDSI community ratified these in the well attended meeting held December 2023. I followed the community contribution process and submitted this request in November, the beginning of this thread.  

When will the vocabulary hot fix be released? :)

**Comment by Alexdavv** [comment]
Hi @MelaniePhilofsky

I understand the pain.
I think the reason it wasn't picked up is that we didn't realize the importance of this first small step of getting the Standard concept for "Multiple race" in the whole pipeline of bringing lots of new concepts to the race Domain. In short, how is this single concept useful if you don't have many others? Because then you can at least play with the existing Standard races in the multiple race model. It wasn't obvious to me, so maybe it wasn’t obvious to others either, and they expected this discussion to be finalized before they could proceed.

The formal answer could be that the forms that you submitted don't change anything in the vocabulary content. But it only adds to the general pain and highlights the complexity of the process and perhaps a lack of guidance/communication/etc in the community contribution process from our end.

But, I still strongly suggest not to rely on the external vocabularies such as PCORNet and go for a new, properly defined concept in the "Race" service vocabulary. Even though it would mean that you can't immediately allow people to use the existing concept_id that could become standard on the next release, because the vocabulary hot fix is not going to happen.

How about two of us sit together to fill in the forms? That could be a first step so that we can plan the vocabularathon, which would bring us closer to the [complex community contribution milestone](https://github.com/OHDSI/Vocabulary-v5.0/wiki/Community-contribution#part-2-complex-use-cases) that would take us quite some effort.

**Comment by Alexdavv** [comment]
We talked to Melanie and want to propose the following:

There are a bunch of reasons why exactly the concept_id = [44814659](https://athena.ohdsi.org/search-terms/terms/44814659) "Multiple race" from the "PCORNet" vocabulary has to become Standard:
- It's already used (although against the conventions) in multiple ETL implementations since there is the [Themis ratified decision](https://github.com/OHDSI/Themis/issues/71#issuecomment-2411803964).
- It's used in PCORI network and is widely known in this respect.
- The community contribution was submitted in time and wasn't formally processed or rejected. The fact that the submitted form doesn't contain any changes comes from the fact that [template 7](https://docs.google.com/spreadsheets/d/1aWYlGvpMiA50Yg-rAPyt4SCK6J8Kvghn/edit?usp=sharing&ouid=114890600408055247292&rtpof=true&sd=true) has a design problem in it and has to be fixed.

To make sure that the promoted concept will be included in all CDM instances and users won't miss it accidentaly, we propose to make the "PCORNet" vocabulary a part of the default Athena distribution (doesn't show up on the [list](https://athena.ohdsi.org/vocabulary/list), included by defauls as many service vocabularies). It won't hurt because it's a tiny vocabulary (81 concepts) and makes sense because it does reflect the structure of another CDM and could help in some interoperability tasks.

If there's no objection, I'll ask @m-khitrun to proceed with the changes in the upcoming August 2025 release.

Tagging @cgreich @aostropolets @TinyRickC137 @vladkorsik @m-khitrun @dimshitc @p-talapova @Piper-Ranallo


Screenshots with the problem in the template:
![Image](https://github.com/user-attachments/assets/d6ef813a-6bcf-4f42-b171-1b42d3c9f272)
![Image](https://github.com/user-attachments/assets/ac5a9150-1ec8-4e01-8aa6-f093f91f2bd3)

**Comment by Piper-Ranallo** [comment]
@Alexdavv -

> But, I still strongly suggest not to rely on the external vocabularies such as PCORNet and go for a new, properly defined concept in the "Race" service vocabulary

I agree strongly with this. 
One of the few remaining RACE  concepts in SNOMED IS 414752008 | Mixed racial group (racial group) |.
What are folks' thoughts about using this as the standard concept for Multiple racial groups?
We could request that SNOMED Interantional add a SYN of 'Multiple racial group'.

Piper

**Comment by TinyRickC137** [comment]
@Alexdavv @MelaniePhilofsky 
the reason the community contribution was not accepted was that we (at least me and Christian, who admitted that a few messages above :) )  were under impression that the vocabularyathon is going to happen between February and August release, and the result of this studyathon is going to be the list of accepted concepts and ETL conventions on how to deal with different Races. There was no intention to harm at all.

Multiple Race from PCORNet should not be standard, and I am surprised @Alexdavv agrees with that. There are multiple reasons why not:
1) PCORNet is known only among PCORNet people. Think about other users. 
2) THEMIS should be adapted when we are ready with the final solution. It is a 'timely' convention. Think of it the other way: if add new concept, would you agree that PCORNet should be standard? All concepts standard from the Race vocabulary except for one?
3) Let's not address the formal part of the matter. Formally it was processed internally, but the classic miscommunication happened. Blame it on me - my apologies.
4) The current implementation implies a curated Race vocabulary. It is controlled by us and can be easily modified, in contrast to basically any other vocabulary as @aostropolets pointed out in the beginning.

Therefore, 
let's proceed with vocabularyathon option and new concepts to solve the problem and stop shoving it under the carpet. With this approach, even if we are unable to resolve the issue, we will definitely add 'Multiple Race' concept during the next release. It sounds like a WG meeting could help.

**Comment by Alexdavv** [comment]
@Piper-Ranallo @TinyRickC137 Thanks! This is a great discussion that could be very apropos before the decision was made and widely promoted by Themis. I'm not sure what it has to do with the under-the-carpet notion since it was widely circulated among all of us. The study-a-thon, in whatever format it is, was planned as the next milestone and doesn't interfere with the need to start using the existing Race concepts in a multiple race format. That's why this request was submitted much in advance, alone, and should not be dropped this way. Now we have to deal with what we have.
